dev-manuals/languages/java/lombok.md

```diff
@@ -0,0 +1,23 @@
+# Lombok
+Because we do not want to write big domain classes with all ther setters, getters and constructors, we want to make the code more readable by using [Lombok](https://projectlombok.org/).
```

Comment:
```suggestion
Because we do not want to write big domain classes with all their getters, setters, and constructors, we want to make the code more readable by using [Lombok](https://projectlombok.org/).
```